db/seeds/dev/stories_seed.js

```diff
@@ -0,0 +1,115 @@
+          Mork the space goblin’s eyes widened at the declaration, and she rushed over to her fellow treasure hunter. 
+          “You mean you found it? We finally found it?”`,
+            `She peered over her partner’s shoulder, and her eyes glittered at the sight. A staff, inlaid with jewels; they had finally found their treasure.
+          The staff was frickin’ happy, but as well as sad, because they had to share the treasure.
+          It’s hard to figure out how to share treasure when it is a living breathing duck. they had to bust out the knives.
+          For anything, the shields were holding, but the dials showed that they would soon fail if the onslaught did not subside. 
+          Drastic action needed to be taken to preserve the lives of the crew. The pilot threw the ship into hyperdrive, knowing 
+          that they were not ready and not everyone might come through the jump alive`,
+            `So everyone put on their seat belts as they all said their prayers.  She was in the back, and she didn’t want the others to see her fear. 
+           Life flashed before her eyes and she began to cry. She clutched her side where the knife was stuck between her ribs. 
+           Her breathing laboured, she held tightly to the wound, willing her bleeding to slow. Mork gasped, and looked at her partner with pleading eyes, “Why?”
+          Her partner explained, “I do not need to tell you the reasoning, just go eat some ice cream.” “When I want the cream you do what I say” her voice 
+          deepened as she dropped her human form, revealing the much larger terrifying creature that she truly was underneath`
+          ],
+          contributors: [3, 5, 2],
+          is_complete: true,
+        },
+        {
+          id: 4,
+          title: "Pokemon the True Story",
+          prompt: null,
+          created_at: "2020-09-10T22:37:51.103Z",
+          updated_at: "2020-09-10T22:37:51.103Z",
+          contributions: [
+              `Ashe Ketchum was the worst trainer in the world. He wanted to get better so he befriended an real 
+            cheese clown named pikachu. Pikachu was the worst friend he could have ever had. 
+            They began walking to their first challenge. At the challenge they met this cool boy named Allen.`, 
+              `Allen wasn’t just cool bc he had a mustache at age 7, but he was also a centaur. Allen liked to race.
+            The glorious mustached 7 year old centaur wasn’t great, but he tried. One race, he tripped over 
+            his front two legs. He screamed in agony because his worst fears had come true, he lost his marbles.`,
+              `Having lost his marbles he could only rely upon his intuition, a rare energy among the centaurs 
+            a space based power within his mind that was devoid of thoughts and rich with imagination.`
+          ],
+          contributors: [5, 4, 3],
+          is_complete: false,
+        }
+      ]);
+    });
+};
```

Comment:
What an awesome way to preserve the OG Corpses that we made together way back when. I would vote that we keep these in here for deployment.